css/index.css

```diff
@@ -1 +1,440 @@
+  
+  header {
+      width: 100%;
+      display: flex;
+      justify-content: space-around;
+  }
+  
+  header nav {
+      margin-top: 4.5%; 
+      width: 69%;
+      justify-content: space-between;
+      display: flex;
+  }
+  
+  header nav a {
+      text-decoration: none;
+      color: #b0b0b0;
+      border-bottom-color: grey;
+      
+      
+      
+      
+
+  }
+  .main-nav .aservices{
+
+  font-size: 1.6rem;
+  
+ }
+
+ .main-nav .aproduct{
+
+    font-size: 1.6rem;
+    
+ }
+ .main-nav .avision{
+
+    font-size: 1.6rem;
+
+ }
+
+ .main-nav .afeatures{
+
+
+    font-size: 1.6rem;
+ }
+
+ .main-nav .aabout{
+
+
+    font-size: 1.6rem;
+ }
+
+ .main-nav .acontact{
+
+    font-size: 1.6rem;
+ }
+  
+  header nav a:hover {
+      text-decoration: underline;
+      color: black;
+  }
+  
+  header .logocon {
+      margin: 3% 0 0 10.3%;
+  }
+  
+  /* Call To Action (CTA) Styles */
+  
+  .cta {
+      display: flex;
+      margin-top: 8%;
+      
+      flex-wrap: wrap;
+      flex-direction: row;
+  }
+  .ctaimage {
+     justify-content: flex-end;
+
+  }
+  
+  .cta .cta-text {
+      font-size: 7.2rem;
+      text-align: center;
+      letter-spacing: .5rem;
+      display: flex;
+      width :25%;
+      flex-direction: column;
+    
+      /* flex-flow: column; */
+      align-items: flex-start;
+      margin: 0 9.3% 0 0;
+  }
+  
+  .cta  button {
+    justify-content: flex-start;
+      border: 1px double black;
+      font-size: 1.6rem;
+      background: white;
+      width: 20%;
+      height:10%;
+      margin-top: 1%;
+      cursor: pointer;
+      order:3;
+      flex-wrap: wrap;
+
+      
+      /* margin-right: 100% */
+  }
+  
+  .cta button:hover {
+      background: black;
+      color: white;
+  }
+  
+  /* Main Content Styles */
+  
+  .main-content {
+     border-top: 2px solid black; 
+     border-bottom: 2px solid black;
+     margin: 9% 0 4% 0;
+     padding: 4% 0;
+     font-size: 1.6rem;
+     line-height: 1;
+     display:flex;
+     flex-wrap: wrap;
+
+  }
+  
+  .main-content .top-content {
+      display: flex;
+  }
+  
+  .main-content .middle-img {
+      display: block;
+      margin: 4% auto;
+      width:100%;
+  }
+  
+  .main-content .bottom-content {
+      display: flex;
+  }
+  
+  .main-content .bottom-content p {
+      padding-right: 2%;
+  }
+  
+  /* Contact Styles */
+  .contact {
+      font-size: 1.6rem;
+  }
+  
+  .contact p {
+      margin: 2% 0;
+  }
+  
+  /* Footer Styles */
+  
+  footer {
+      text-align: center;
+      margin: 5% 0 3%;
+  }
+/* Tablet View */
+  @media (max-width:800px){
+      
+    header nav {
+        width: 100%;
+        margin: 3% 3% 0px -2%;
+        justify-content: space-around;
+    }
+
+    header{
+        flex-direction: column-reverse;
+        flex-wrap: wrap;
+    }
+
+    header .logocon {
+        align-self: center; 
+        margin: 3% 0px 21px 0px;
+    }
+    .ctaimage {
+        display: none;
+    }
+
+    .cta {
```

Comment:
Remember, you need to apply display: flex to parent level elements to control the children.